### PR TITLE
1346 Make facet buttons blue

### LINKED ISSUE #1340: Map facet buttons blue not red
See "maps"

Screenshot 2024-08-30 at 16 16 16

### PR DESCRIPTION
**Pull request by peetucket**
Fixes #1340 

Looks like this in PR:

Screenshot 2024-08-30 at 3 37 31 PM
Screenshot 2024-08-30 at 3 37 35 PM

### CODE REVIEW

File: app/assets/stylesheets/earthworks.css [hunk 1]
```diff
@@ -313,3 +313,21 @@ label.toggle-bookmark {
   --bs-link-color-rgb: var(--stanford-digital-blue-dark-rgb);
   text-decoration: underline;
 }
+
+.constraint-value.btn-outline-secondary {
```
Comment (jcoyne) [inline app/assets/stylesheets/earthworks.css:317]: Secondary is defined as red in the component library.  Can you change the class to be btn-outline-secondary instead?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/earthworks.css (hunk 1 above)
Comment (jcoyne) [reply app/assets/stylesheets/earthworks.css:317]: This is in https://github.com/projectblacklight/blacklight/blob/a4ec2d85eeb8d61bd2d462ccdfb5de1c50dba27d/app/components/blacklight/constraint_layout_component.html.erb#L2

Comment (jcoyne) [reply app/assets/stylesheets/earthworks.css:317]: That is provided by https://github.com/projectblacklight/blacklight/blob/a4ec2d85eeb8d61bd2d462ccdfb5de1c50dba27d/app/components/blacklight/constraint_component.rb#L7
and https://github.com/projectblacklight/blacklight/blob/a4ec2d85eeb8d61bd2d462ccdfb5de1c50dba27d/app/components/blacklight/constraints_component.rb#L26C20-L26C46
Which you can configure in https://github.com/projectblacklight/blacklight/blob/a4ec2d85eeb8d61bd2d462ccdfb5de1c50dba27d/lib/blacklight/configuration.rb#L187

**Comment by jcoyne** [comment]
This seems to be in conflict with what is asked for in https://github.com/sul-dlss/earthworks/issues/1355

**Comment by peetucket** [comment]
> This seems to be in conflict with what is asked for in #1355

Indeed it does -- I didn't see the figma design when doing this ticket.  Closing this PR and the associated ticket.

**Pull request closed without merging**